web-app/src/app/services/feeds/utils.ts

```diff
@@ -50,9 +50,9 @@ export type AllDatasetType =
   | undefined;
 
 export function getLocationName(
-  locations: EntityLocations | undefined,
+  locations: EntityLocations | undefined | null,
 ): string {
-  if (locations === undefined) {
+  if (locations === undefined || locations === null) {
```

Comment:
Can't we use `locations == null` to cover both cases? Why do we use a strict equality operator?